src/v0/destinations/ga4/utils.js

```diff
@@ -432,11 +433,39 @@ const prepareUserProperties = (message, piiPropertiesToIgnore = []) => {
   return validatedUserProperties;
 };
 
+/**
+ * Returns user consents
+ * Ref : https://developers.google.com/analytics/devguides/collection/protocol/ga4/reference?client_type=gtag#payload_consent
+ * @param {*} message
+ * @returns
+ */
+const prepareUserConsents = (message) => {
```

Comment:
Is this not mandatory / recommended for EU customers? Because I do not see any default values being considered. Would need a go-ahead from @krishna2020 or @vthopu.

Reply:
We are providing feature to send consents, so we are good. As it's optional field and there's no value like UNSPECIFIED, we can't take call on behalf of customer - so it's on EU customer end to instrument it mandatorily.